docs/configuring.md

```diff
@@ -85,6 +85,26 @@ particular set of drivers to try with
 `jupyter-lite.json#jupyter-config-data/contentsStorageDrivers`. See more about
 [local storage drivers](#local-storage-drivers).
 
+### Accessing local files with the `jupyterlab-filesystem-access` extension
+
+The `jupyterlab-filesystem-access` extension allows accessing local files using the
+(non-standard) Web Browser File System Access API.
+
+The extension is available on [GitHub][filesystem-access-github] and published to
+[PyPI][filesystem-access-pypi] . It is compatible with both JupyterLab and JupyterLite.
+
+See the section below to learn how to add extensions to your deployment.
+
+```{warning}
+This extension is compatible with Chromium-based browsers only (for now).
+
+See the [MDN][mdn] documentation for more information.
+```
+
+[mdn]: https://developer.mozilla.org/en-US/docs/Web/API/File_System_Access_API
+[filesystem-access-pypi]: https://pypi.org/project/jupyterlab-filesystem-access/
+[filesystem-access-github]: https://github.com/jtpio/jupyterlab-filesystem-access
```

Comment:
Will wait for https://github.com/jupyterlab-contrib/jupyterlab-contrib.github.io/issues/31 to be completed to update to the new URL: https://github.com/jupyterlab-contrib/jupyterlab-filesystem-access

Reply:
Done.